lib/__tests__/cookieJar.spec.ts

```diff
@@ -1203,6 +1203,48 @@ it('should fix issue #282 - Prototype pollution when setting a cookie with the d
   expect('/notauth' in pollutedObject).toBe(false)
 })
 
+it('should fix issue #154 - Expiry should not be affected by creation date', async () => {
+  // setting the time to zero here just to make it obvious that the expiry time is updated
+  jest.useFakeTimers({ now: 0 })
```

Comment:
Do we need to use real timers after the test, or is this scoped to the test?

Reply:
are you asking in case we need to restore the timers after the test run?  because the test suite relies on mocking out the timers but this is the only one that sets `now` to 0.

Reply:
@wjhsf i've enabled the fake timers globally in the jest config since we have no tests that rely on a real timers in our jest suite.